WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -319,6 +319,9 @@
       end
 
       it 'cannot pass a non-delegate as delegate' do
+        # Update competition to be in the future
+        competition.update_attributes(start_date: 1.day.from_now.strftime("%F"), end_date: 1.day.from_now.strftime("%F"))
```

Comment:
BTW, these things are good candidates for FactoryGirl traits: `:past`, `:future`, and maybe `:ongoing`